msteams-platform/toolkit/tools-prerequisites.md

```diff
@@ -68,10 +68,10 @@ Ensure that you have the following accounts before you start building your Teams
 > [!NOTE]
 > To build Teams apps, you must have a work or school [Microsoft 365 account](#microsoft-365-developer-program). Microsoft 365 personal account can't be used.
 
-If you don't have any, to create a developer account, sign up for a Microsoft 365 developer program subscription. The subscription is free for 90 days and continues to renew as long as you're using it for development activity.
-
 If you have a Visual Studio Enterprise or Professional subscription, both programs include a free Microsoft 365 [developer subscription](https://aka.ms/MyVisualStudioBenefits). It's active as long as your Visual Studio subscription is active. For more information, see [Microsoft 365 developer subscription](https://developer.microsoft.com/microsoft-365/dev-program).
 
+If you don't have any Microsoft 365 tenant, you might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](https://aka.ms/m365devprogram); for details, see the [FAQ](/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-). Alternatively, you can sign up for a [1-month free trial](https://www.microsoft.com/microsoft-365/try) or [purchase a Microsoft 365 plan](https://www.microsoft.com/microsoft-365/business/compare-all-microsoft-365-business-products-g).
```

Comment:
These are two related independent clauses rather than separate thoughts, so I prefer that we keep the semicolon here. The details/information is specifically related to whether they qualify for a subscription. Also, "for details" is a shorter, more readable phrase, and conveys that details about program qualifications are found in the FAQ.

If this is a blocking issue for merging, I'm okay with the change but I prefer the original construction if possible. This is also the boilerplate text that we've used in other content sets that reference the Dev Program. Thanks!